lib/consensus/voting_policy.go

```diff
@@ -65,16 +65,11 @@ func (vt *ISAACVotingThresholdPolicy) Threshold(state ballot.State) int {
 	v := float64(vt.validators) * (float64(t) / float64(100))
 	threshold := int(math.Ceil(v))
 
-	// in SIGN state, proposer assumes to say VotingYES
-	if state == ballot.StateSIGN {
-		threshold = threshold - 1
+	if threshold < 0 {
```

Comment:
Let's assume that there are 1000 nodes and threshold is 66 percent.
When 65 of 99 nodes send Yes, the `Threshold` method return 659.  Ceil(999 * 66 / 100 ).
In this case, the consensus pass when got 659 yes. But it should not pass until got 665 yes.

Did you consider this?

Reply:
@kfangw 
The Threshold method return 660(1000 * 66/100). And threshold(66) means 66%. Not 2/3(66.66666).
So failure in 660 and pass in 661.
I added unit test for checking threshold :)

Reply:
We need 2/3 so we should think the other way to setting threshold.

Reply:
I confused ^^;